### ISSUE TITLE
[Improving] Not meeting use case

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-planning-and-implementation#62]
Re: [Improving the Accessibility of Your Website](//w3c.github.io/wai-planning-and-implementation/improving.html)

I put in the [survey](https://www.w3.org/2002/09/wbs/35532/April2016Aprov1/results#xq6) that I had concerns, but did not object to publishing this version and would abstain from voting. Shadi & Brent asked me to give some more specific examples for EOWG to consider if we want to publish as is or revisit &mdash; so here you go :)

EOWG defined this document to address the use case: "Help! I need to fix accessibility on my website now!" I think this iteration of the document is too wordy and has too much extraneous info based on that goal.

One major issue is that in the current iteration, many of the things that are presented as imperatives, should be suggestions or even just ideas to consider. For example,
- "Evaluate with real users" -- that's a _major_ undertaking for many project teams who haven't done anything like that before, not a "help, now" tip.

Just a couple examples of where the current draft doesn't seem to meet the use case:
- "Before assessing, fix any known barriers" While this is important advice for before doing usability testing, it maybe not the best approach for everyone in this use case. It would mean: fixing, then evaluating, then fixing again. In some situations, it might best to figure out the issues first, then fix them.
- "Evaluate using a standardized approach. … WCAG-EM" They might want to skim WCAG-EM to see if any info is useful, but they probably do not need to spend effort on formalizing a standardized approach yet.

Examples of unnecessary wordiness:
- First paragraph under "Understanding the issue" can probably be just one sentence.
- "Define the scope" can be tersified to probably ¼ the length. (Also: Does this overlap with prioritize section too much? And "journeys" doesn't seem like the right word.)

In addition, I think there are lots of minor things, such as:
- "Creating a graph…" is that really an effective use of time?
- "Discussion with everyone involved can help you quickly determine possible solutions…" – In some orgs, just finding a meeting time with "everyone involved" isn't quick. "Ensure that whoever is responsible… is involved." Can suggest it, but not "ensure" as that's often not possible.
- "Perform detailed review" – first para not clear enough and too wordy.
- "For example, issues in templates that are shared between many pages, will have a broad impact and issues in key pages, such as the home page or shopping cart checkout, may have a specific but significant impact." Unclear
- Under "Implement repairs", in the first two tips I noted "not clear" and "huh?"
- "There are several things you can do to help your organization create more accessible websites as standard." "as standard" too easily confused with "accessibility standards such as WCAG" in this context.

---

Basically I think this doc should cover:
- Get a basic understanding of accessibility
- Get the skills and tools (eval and authoring) needed to address accessibility
- Set the target (e.g., WCAG Level AA) and scope (parts of website)
- Evaluate to identify the issues
- Prioritize the fixes – by barriers, by area (e.g., [previous version](https://www.w3.org/WAI/EO/Drafts/retrofit/Overview.html#prior))
- Tips for optimizing the work – distributing tasks, clarifying requirements, validating solutions before implementing them (e.g., some of [previous version](https://www.w3.org/WAI/EO/Drafts/retrofit/Overview.html#retro))
- (Longer term: point to Planning & Managing)

**Comment by susanatx** [comment]
Note to Shadi and others: My promised breakdown of concerns will come this weekend. I've just now organized my thoughts and must clock out for the night. I always overestimate my bandwidth to deliver what I promise.

**Comment by susanatx** [comment]
_"EOWG defined this document to address the use case: "Help! I need to fix accessibility on my website now!" I think this iteration of the document is too wordy and has too much extraneous info based on that goal."_

Absolutely agree here 

_"Before assessing, fix any known barriers" _

While this is important advice for before doing usability testing, it maybe not the best approach for everyone in this use case. It would mean: fixing, then evaluating, then fixing again. In some situations, it might best to figure out the issues first, then fix them."

Along these lines, I don't think it fits with this resource. If we're helping them fix things now, then either they know the barriers which is why they're looking for some immediate fixes they can make or if they don't know, then let's direct them to how to evaluate. "Fix any known," doesn't seem applicable for this this purpose.

I agree with Shawn here as well:
_"Examples of unnecessary wordiness:

First paragraph under "Understanding the issue" can probably be just one sentence.
"Define the scope" can be tersified to probably ¼ the length. (Also: Does this overlap with prioritize section too much? And "journeys" doesn't seem like the right word.)"_

Overall, this resource should be much more accessible (not in the tech sense,) engaging, and something users would want to share, link to, and use as exactly what we say it is - how to fix now. So much of our content is very deep, wordy, and often difficult to sort out, this is a chance to provide a great resource to those looking for handy, quick fixes. It could also serve as a great gateway for people to dig deep in our site for more comprehensive information.

Suggestions for overall improvement of resource:

-A more engaging title the makes its purpose clear from the start, e.g "Quick Fixes (or tips) for Improving the Accessibility of Your Website Now," (less words would be better.) The current title is too similar to other resources and doesn't convey this page's purpose well. Let's make it clear these are quick fixes and things that can be done right away. 
- Cut 'Understand the Issue" all together. Link them to our broader P & M Web A11y guide and don't spend space reiterating high level info that's stated in many other documents.

-Move the list of links to the bottom as "other resources/references". Why send them to other sites as soon as they come to this page?

-Cut down all sections as much as possible - more bullets, less paragraphs. This should be something that can quickly be scanned.

-Move "Define your Target" into "Define the Scope." Choosing your target compliance level is part setting the project's scope.

-Give more concrete examples and tips in "Implement Repairs." _How_ can they implement them? If they've covered the steps before this one then they know they need to fix forms, keyboard control, etc. We need to give them help to  learn how to do it or we aren't telling them much about how to do this now. Don't link to "How People with Disabilities..." That's the understanding of accessibility, not fixing it.  Link to "Tips for Getting Started" instead. This give them examples of why something could be wrong, why it's important, and ideas of how to fix it. That resource also links to QuickRef and the user stories. Those should be additional info for those who need to dig deeper past what's on this page. They don't represent what should be the use case.

I think that's plenty of words from me for now.

**Comment by nitedog** [comment]
Addressed ([commenters notified](https://lists.w3.org/Archives/Public/wai-eo-editors/2016Apr/0036.html))
